cpp/src/arrow/filesystem/s3fs.cc

```diff
@@ -2986,9 +2994,7 @@ Status InitializeS3(const S3GlobalOptions& options) {
   return Status::OK();
 }
 
-Status EnsureS3Initialized() {
-  return EnsureAwsInstanceInitialized({S3LogLevel::Fatal}).status();
-}
+Status EnsureS3Initialized() { return EnsureAwsInstanceInitialized().status(); }
```

Comment:
How about passing `S3GlobalOptions::Defaults()` here instead of defining `EnsureAwsInstanceInitialized(void)` because `EnsureAwsInstanceInitialized(void)` will not be reused?

```suggestion
Status EnsureS3Initialized() { return EnsureAwsInstanceInitialized(S3GlobalOptions::Defaults()).status(); }
```

Reply:
Sure, that looks like a nice simplification.